xml/admin_ceph_gateway.xml

```diff
@@ -288,12 +288,16 @@ for bucket in conn.get_all_buckets():
      you more about its command line options.
     </para>
     <para>
-     The package is included in the 'Public Cloud' module for &sle; 12 SP3.
+     The package is included in the 'Public Cloud' module for &sle; 12 SP3 and
+     &sle; 15 SP1.
```

Comment:
since 12 SP3 (it's in SP4, SP5 as well) - and we can generalize for 15, it's in GA, SP2, what about just saying "SLES 12 since SP3 and SLES 15" ?